### DISCUSSION TITLE
Arbitrary groups

### DISCUSSION THREAD

**Discussion by FOSSgeek** [tailwindlabs/tailwindcss#10113] [Help]
what exactly https://tailwindcss.com/docs/hover-focus-and-other-states#arbitrary-groups is? can someone give me a practical interactive example where arbitrary groups is useful at all?

**Comment by wongjn** [comment]
It is useful when you need to apply styling to child elements based on a parent element and where the built in `group-` variants do not cover the selector you'd like to write. For example:

```html
<div class="group is-published">
  <div class="hidden group-[.is-published]:block">
    Published
  </div>
</div>
```
